src/__tests__/internal/Pagination.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import Pagination from '../../components/internal/Pagination';
+
+    // Shallow Rendering
+    // https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md
+describe('Pagination - Shallow Rendering', () => {
+    it('to have the correct number of Pagination classes', () => {
+      const wrapper = shallow(
+        <Pagination
+          onChange={() => alert('Switch Page...')}
+          page={3}
+          pagesize={4}
+          numPages={6}
+        />,
+      );
+      expect(wrapper.find('.mars')).to.have.length(2);
+    });
+});
+
```

Comment:
Add tests for the following:
- pagination buttons work, i.e. mock the required state (pages, page size etc.) to allow the next/prev buttons to work
- the next button won't go to the next page if it is on the last page already (same for prev button on the 1st page)